### PR TITLE
feat(sourcemaps): Add basic source maps upload functionality

### PR DESCRIPTION
**Pull request by Lms24**
* make Sentry CLI calls
* adds default options
* add small node sample app to test sourcemaps upload

### CODE REVIEW

File: packages/playground/src/smallNodeApp.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Sentry = require("@sentry/node");
+const { RewriteFrames } = require("@sentry/integrations");
+
+// import * as Sentry from '@sentry/node';
+// import { RewriteFrames } from "@sentry/integrations";
```
Comment (lforst) [inline packages/playground/src/smallNodeApp.js:5]: Is this still needed?

File: packages/playground/config.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "configFile": "./sentry.properties",
```
Comment (lforst) [inline packages/playground/config.json:2]: Is this still needed? As far as I can see we don't have a properties file

File: packages/playground/vite.config.smallNodeApp.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// @ts-check
+import { sentryVitePlugin } from "@sentry/unplugin";
+import { defineConfig } from "vite";
+import * as path from "path";
+import placeHolderOptions from "./config.json";
```
Comment (lforst) [inline packages/playground/vite.config.smallNodeApp.js:5]: do we want to use the placeholderoptions in this file?

**Review by lforst — APPROVED** [review]

File: packages/playground/src/smallNodeApp.js (hunk 1 above)
Comment (Lms24) [reply packages/playground/src/smallNodeApp.js:5]: nope :D

File: packages/playground/config.json (hunk 2 above)
Comment (Lms24) [reply packages/playground/config.json:2]: Nope :D

File: packages/playground/vite.config.smallNodeApp.js (hunk 3 above)
Comment (Lms24) [reply packages/playground/vite.config.smallNodeApp.js:5]: Nope :D

**Pull request merged**